### ISSUE TITLE
Vídeos em HTML5

### ISSUE THREAD

**Issue by belisards** [grama-cc/nuvela#3]
Oi, agora a dúvida: há como puxar os vídeos do youtube em html5? Aqui no Iceweasel (Firefox) a Nuvela não rodou (ficou naquela estática da TV) até que eu habilitasse o vídeo. Além disso, o Flash já foi descontinuado, ne?
Seria tranquilo uma alteração dessa?

parabéns mais uma vez pelo trabalho!

**Comment by euharrison** [comment]
@belisards obrigado pelos feedbacks, vamos investigar esses dois itens.
se tiver mais sugestões, é só falar!

**Comment by sulram** [comment]
se aqueles jw players da vida tiverem esse controle de começar o video de um ponto específico, talvez seja uma possibilidade... de repente resolve o problema com as ads tb

**Comment by sulram** [comment]
será que esse aqui?

http://mediaelementjs.com/examples/?name=youtube

**Comment by sulram** [comment]
no meu firefox a NUVELA abre com player HTML5 do Youtube...

**Comment by euharrison** [comment]
Vamos testar esse player, mas pelo que vi rapidamente ele usa o próprio embed do youtube, só cria uma camada de controles por cima.

Em teoria o próprio embed do youtube que decide qual é o melhor formato a ser exibido. Um bom teste é tentar acessar o mesmo video diretamente pela url, no link de ver o video completo.

Mas vamos esperar os primeiros testes do player, o @edinhoo vai trazer essa informação para nós e aí poderemos discutir em cima

**Comment by sulram** [comment]
beleza, @euharrison 
depois eu reparei que ele só joga os controles por cima do player do youtube, talvez a boa seja continuar com youtube, mas ver como forçar o player html5.

mas @belisards, vamos focar nos browsers mais modernos e mais populares

**Comment by belisards** [comment]
Certo, mas o.iceweasel eh fundamentalmente o firefox, vc sabe né? Acho q
difícil q seja algo com ele especificamente mas vamos ver...
Em 23/03/2016 18:38, "Marlus Araujo" <notifications@github.com> escreveu:

**Comment by euharrison** [comment]
fizemos os testes com o player do mediaelements e parece que ele consegue esconder os anúncios, mas ainda não identificamos como ele faz isso. Vamos estudar o source deles para tentar aplicar o mesmo conceito

**Comment by euharrison** [comment]
@belisards fiz uns testes aqui e parece estar funcionando bem no firefox. Quem provê o vídeo em HTML5 ou Flash é o próprio Youtube, não temos controle sobre isso.

Uma coisa que pode estar confundindo é o fato da Nuvela pausar quando o browser não está em foco, é um recurso que usamos caso alguém dê "alt+tab", o audio não ficará incomodando a mudança de foco. 

Poderia reconfirmar para mim se este ainda é um bug?
